napalm_yang/translator.py

```diff
@@ -93,6 +96,7 @@ def _translate_list(self, attribute, model, mapping, translation, other):
         # Saving state to restore them later
         old_parent_key = self.keys["parent_key"]
         old_parent_bookmark = self.bookmarks["parent"]
+        old_extra_vars = copy(self.extra_vars)
```

Comment:
Could you use `deepcopy` instead? Like here:

https://github.com/napalm-automation/napalm-yang/blob/c6e30fd2afac6c133ef93e11d456d585d1e2f716/napalm_yang/parser.py#L131